AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts

```diff
@@ -362,10 +350,8 @@ export class AssignJudgeComponent extends BookingBaseComponent implements OnInit
             if (this.hearingService.canAddJudge(judge.username)) {
                 judge.is_judge = true;
                 judge.case_role_name = 'Judge';
-                if (this.useV2Api) {
-                    judge.case_role_name = null;
-                    judge.hearing_role_code = Constants.HearingRoleCodes.Judge;
-                }
+                judge.case_role_name = null;
```

Comment:
this is now overwriting the line above it. So `judge.case_role_name = 'Judge'` can be removed

Reply:
done